invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/Home/components/widgets/ACQRequestsCard/ACQRequestsCard.js

```diff
@@ -35,14 +35,14 @@ export default class ACQRequestsCard extends Component {
     );
   };
 
-  _render_card = data => {
+  render_card = data => {
```

Comment:
```suggestion
  renderCard = data => {
```
There were a few more `render_card` as well.